packages/components/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 -   `InputControl`: Fix internal `Flex` wrapper usage that could add an unintended `height: 100%` ([#46213](https://github.com/WordPress/gutenberg/pull/46213)).
 -   `Navigator`: Allow calling `goTo` and `goBack` twice in one render cycle ([#46391](https://github.com/WordPress/gutenberg/pull/46391)).
 -   `Modal`: Fix unexpected modal closing in IME Composition ([#46453](https://github.com/WordPress/gutenberg/pull/46453)).
+-   `FocalPointPicker`: Reset `margin-bottom` of `UnitControl` to align vertical position ([#46631](https://github.com/WordPress/gutenberg/pull/46631)).
```

Comment:
Is this PR a bug fix? Maybe it is 'internal' or 'enhancement'.

Reply:
I would classify it as a bug fix.

Reply:
I have moved the changelog entry to the `block-editor` package.